include/mbedtls/ecp.h

```diff
@@ -276,15 +276,15 @@ mbedtls_ecp_group;
 
 #if !defined(MBEDTLS_ECP_FIXED_POINT_OPTIM)
 /*
- * Trade memory for speed on fixed-point multiplication.
+ * Trade ROM usage for speed on fixed-point multiplication.
```

Comment:
Minor: It's probably better to call it "code size" - as the code may be stored in a variety of places, quite often flash which is not ROM. (That's partially a pre-existing issue, as there are other places in the documentation where we write ROM, but people have been pointing out that it's not correct, so let's try to improve at least in new documentation.)

Note: this holds here as well as in the ChangeLog and migration guide.

Reply:
fixed